tsconfig.legacy.json

```diff
@@ -1,22 +1,23 @@
+
```

Comment:
This legacy file was only needed for @ngrx/core to preserve deep imports. You should not need a legacy build step.